app/assets/stylesheets/ama_layout/layout_components/legends.scss

```diff
@@ -0,0 +1,132 @@
+.legend{
+  &__header{
+    background-color: $cerulean;
+    color: $white;
+    padding: $base-padding*0.6;
+  }
+
+  &__content-container{
+    background-color: $white;
+    padding: $base-padding*0.6;
+
+    &.visible{
+      display: block;
+      clear: both;
+    }
+  }
+
+  &__keys{
+    list-style: none;
+    margin-bottom: 0;
+  }
+
+  &__key-identifier{
+    @include clearfix;
+    margin-top: $base-margin*0.4;
+  }
+
+  &__icon{
+    height: 12px;
+    width: 12px;
+    float: left;
+    margin-right: $base-margin*0.45;
+    margin-top: $base-margin*0.2;
+    border-radius: 100%;
```

Comment:
@kaytcampbell yeah i realized that after my comment, which is why i just left it as is